plugin/trino-spooling-filesystem/src/main/java/io/trino/spooling/filesystem/FileSystemSpoolingConfig.java

```diff
@@ -81,6 +83,19 @@ public FileSystemSpoolingConfig setLocation(String location)
         return this;
     }
 
+    public Layout getLayout()
+    {
+        return layout;
+    }
+
+    @Config("fs.layout")
+    @ConfigDescription("Spooled segments filesystem layout")
+    public FileSystemSpoolingConfig setLayout(Layout layout)
+    {
+        this.layout = layout;
+        return this;
+    }
```

Comment:
why to allow user to configure that?
Seems like it depends on what FS you use under the hood. If S3 - got with this layout. If GCS go with that.
Maybe MINIO vs S3 is the reason - but not sure if that justifies extra config option.

Reply:
I'd say that this is a deployment option